application/test_cloud.py

```diff
@@ -104,6 +104,17 @@
     remote.speedDistribution(identifier, project_path)
     raw_input('Generating Speed Distribution, please wait...\n Press Enter to Continue')
 
+    ###########################################################################
+    # Run Compare Routes
+    ###########################################################################
+    identifier2 = None # input an identifier to compare current project to 
+    remote.compareSpeeds(identifier, [identifier, identifier2], ['label1', 'label2'], project_path, False)
```

Comment:
Do we want to put this in an "if identifier2:" or will the server/cloud_api handle it?